omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/DelegatingCrudResourceTest.java

```diff
@@ -172,5 +175,74 @@ public void convert_shouldConvertASimpleObjectThatIncludesAUuid() {
 		Location location = resource.convert(so);
 		Assert.assertEquals(uuid, location.getUuid());
 	}
-	
+
```

Comment:
FWIW, I was forgetting if there was a straightforward way to test with a user that *does not* have the set audit data privilege.